fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -310,7 +310,14 @@ public ShowResultSet visitShowStatement(ShowStmt statement, ConnectContext conte
         @Override
         public ShowResultSet visitShowMaterializedViewStatement(ShowMaterializedViewsStmt statement, ConnectContext context) {
             String dbName = statement.getDb();
-            Database db = GlobalStateMgr.getCurrentState().getLocalMetastore().getDb(dbName);
+            String catalogName = statement.getCatalogName();
```

Comment:
why do we need to update/fix `ShowExecutor` for `show materialized views`?

Reply:
because ShowExecutor handles the SHOW command including who materialised views.

Reply:
oh. Somehow I thought `ShowExecutor` is for `show backends` for the backend/executor nodes. This is for executing `show` statements.